unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -43,7 +43,7 @@ export class CreatorLockForm extends React.Component {
       // Unlimited keys is represented as zero, so when we load a lock that
       // has `maxNumberOfKeys` set to zero, convert it to the infinity symbol
       // before displaying it.
-      if (maxNumberOfKeys === 0) {
+      if (maxNumberOfKeys === UNLIMITED_KEYS_COUNT) {
```

Comment:
Actually I think we should remove this line. `maxNumberOfKeys` can (and should) stay `-1`.

Reply:
Also, that comment will have to change if we're no longer using 0 to represent infinity